en/option-gl/component/globe.md

```diff
@@ -2,75 +2,210 @@
 {{ target: component-globe }}
 
 # globe(Object)
-
-地球组件。组件提供了地球的绘制以及相应的坐标系，开发者可以在上面展示三维的散点图，气泡图，柱状图，飞线图。
+Provides settings for the globe component that allows to render three-dimensional scatter chart, bar chart or lines.  
 
 ## show(boolean) = true
 
-是否显示地球组件。
-
-{{ use: partial-zlevel }}
-
-{{ use: partial-viewport }}
+Displays globe when true.
 
-## globeRadius(number) = 100
-地球的半径。单位相对于三维空间，跟 [viewControl.distance](~globe.viewControl.distance) 相关。
+## globeRadius (number) = 100
+Globe radius in the three-dimensional viewport. Globe distance can be changed via [viewControl.distance] (~ globe.viewControl.distance) settings.
 
-{{ use: partial-environment }}
+{{ use: partial-environment(
+    componentType="globe",
+    componentName="Globe"
+) }}
 
 ## baseTexture(string|HTMLImageElement|HTMLCanvasElement|EChartsInstance)
 
-地球的纹理。支持图片路径字符串，图片或者 Canvas 的对象。
+Globe texture, supports image path, HTML image element, HTML canvas element or another echarts instance. 
 
-也支持直接使用 echarts 的实例作为纹理，此时在地球上的鼠标动作会跟纹理上使用的 echarts 实例有联动。
+Globe texture becomes fully interactive if another echarts instance is used. 
 
-示例：
+Example:
 ```js
-// 使用地球的纹理图片
+// use the Earth texture image
 baseTexture: 'asset/earth.jpg'
 
 
-// 使用 echarts 绘制世界地图的实例作为纹理
+// Example echarts drawing the world map as a texture
 var canvas = document.createElement('canvas');
-var mapChart = echarts.init(canvas, null, {
+var Mapchart = echarts.init (canvas, null, {
     width: 4096, height: 2048
 });
 mapChart.setOption({
     series : [
         {
             type: 'map',
             map: 'world',
-            // 绘制完整尺寸的 echarts 实例
+            // Draw full size echarts examples
             top: 0, left: 0,
             right: 0, bottom: 0,
             boundingCoords: [[-180, 90], [180, -90]]
         }
     ]
 });
 ...
-baseTexture: mapChart
+BaseTexture: mapChart
 
 ```
 
 ## heightTexture(string|HTMLImageElement|HTMLCanvasElement)
+Height texture can be provided to render globe surface in three dimensions. Echarts uses [bump mapping] (https://zh.wikipedia.org/wiki/%E5%87%B9%E5%87%B8%E8%B4%B4%E5%9B%BE) which simulates bumps and wrinkles on the surface of an object. Below are two examples, one using heightTexture and another one without heightTexture:  
```

Comment:
* No whitespace between `]` and `(`.
* Should link to the English version of Wikipedia: https://en.wikipedia.org/wiki/Bump_mapping